terraform/aws/components/fake_mesh/data_remote_state_account.tf

```diff
@@ -0,0 +1,9 @@
+data "terraform_remote_state" "account" {
+  backend = "s3"
+  
+  config = {
+    bucket = var.tf_state_bucket
+    key = "${var.project}-account-account.tfstate"
```

Comment:
other files use `${var.project}-${var.environment}`, is above correct?

Reply:
yes, this is correct, account is specific, where both the environment and component are named 'account' as it should exists only in one copy

Reply:
Account may be a special case. @LukaszKwasniewski ?